### ISSUE TITLE
Документация для ssa with weights

### ISSUE THREAD

**Issue by asl** [asl/rssa#219]
Собственно, сабж.

1. Документация
2. Примеры

**Comment by furiousbean** [comment]
Первая попытка что-то сделать.

https://github.com/furiousbean/rssa/commit/3fa06c4e084bd4b08d75f2661bd378091373e60d

Пока не хватает примера в документации.

**Comment by asl** [comment]
Да, хорошо. Но хотелось бы еще теста не только на cadzow. И примера без cadzow :)

**Comment by asl** [comment]
Правда, не знаю, насколько это реально.

**Comment by StasToken** [comment]
а почему вы пример в wiki не напишите? было бы удобнее мне кажется. 
PS: спасибо за пакет.
